### ISSUE TITLE
Formats and CORS

### ISSUE THREAD

**Issue by marcoscaceres** [immersive-web/model-element#15]
Need to clarify that 3D resources can fetch resources, and as such need to be subject the document's CORS policy (probably "media-src"). However, we need to clarify what this means in relation to, say, "img-src", for example... as models can load png/jpg textures.

**Comment by domenic** [comment]
This issue seems to be about CSP, not CORS?

**Comment by marcoscaceres** [comment]
Ah whoops, yes. Acronym soups getting all mixed up in my head. Fixed the title and OP.

**Comment by domenic** [comment]
> However, we need to clarify what this means in relation to, say, "img-src", for example... as models can load png/jpg textures.

I don't know the answer myself, but I guess we can look at the precedents from CSS and SVG.

**Comment by cabanier** [comment]
> Ah whoops, yes. Acronym soups getting all mixed up in my head. Fixed the title and OP.

Browser generally frown upon subresource fetches. The model element should fetch only 1 file and not reach out later for additional ones. SVG-as-a-image was changed to no longer allow fetches.

**Comment by zachernuk** [comment]
I agree that model is not the place to compose sub-resource fetches, and believe the explainer reflects that. Do we keep this around to try to put it into spec (and then _HTML_ spec) text, or do we close things like this? @marcoscaceres what is the most constructive way to approach things like this?
/facetoface to confirm that nobody's on the fence about this and we have enough clarity in any documentation about it

**Comment by zachernuk** [comment]
(we can also discuss the fetch destination - it looks like visionOS currently uses a `Sec-Fetch-Dest` of `model`, which seems like a reasonable thing to use, based on the list of destinations listed here:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Sec-Fetch-Dest

**Comment by AdaRoseCannon** [comment]
Resolution: Everyone seems cool with no sub resource fetching
